### PR TITLE
Correct the comment  message

### PR DESCRIPTION
**Pull request by donggangcj**
Fix the comment error

### CODE REVIEW

**Comment by donggangcj** [comment]
I have registered.

File: src/app/backend/resource/common/resourcechannels.go [hunk 1]
```diff
@@ -374,7 +374,7 @@ func GetPodListChannelWithOptions(client client.Interface, nsQuery *NamespaceQue
 	return channel
 }
 
-// ReplicationControllerListChannel is a list and error channels to Nodes.
+// ReplicationControllerListChannel is a list and error channels to replication.
```
Comment (maciaszczykm) [inline src/app/backend/resource/common/resourcechannels.go]: It's incorrect.

File: src/app/backend/resource/common/resourcechannels.go [hunk 2]
```diff
@@ -486,7 +486,7 @@ func GetReplicaSetListChannelWithOptions(client client.Interface, nsQuery *Names
 	return channel
 }
 
-// DaemonSetListChannel is a list and error channels to Nodes.
+// DaemonSetListChannel is a list and error channels to damonSet.
```
Comment (maciaszczykm) [inline src/app/backend/resource/common/resourcechannels.go]: Here too.

File: src/app/backend/resource/common/resourcechannels.go [hunk 3]
```diff
@@ -581,7 +581,7 @@ func GetCronJobListChannel(client client.Interface, nsQuery *NamespaceQuery, num
 	return channel
 }
 
-// StatefulSetListChannel is a list and error channels to Nodes.
+// StatefulSetListChannel is a list and error channels to statefulset.
```
Comment (maciaszczykm) [inline src/app/backend/resource/common/resourcechannels.go]: Here too.

**Pull request closed without merging**